azure-core/azure/core/configuration.py

```diff
@@ -24,9 +24,11 @@
 #
 # --------------------------------------------------------------------------
 
+#pylint: disable=too-few-public-methods
 
 class Configuration(object):
 
+    #pylint: disable=too-many-instance-attributes
```

Comment:
Just noting that this pragma is already used ~7 times outside this PR already, perhaps it's also worth considering raising the pylint `max-attributes` to reflect the specific needs of our codebase.